codeJE/dpl-config.json

```diff
@@ -445,10 +445,10 @@
     },
     "jet-validation-track-collision-qa": {
         "ptLow": "0.15",
-        "ptUp": "100",
+        "ptUp": "1e15",
         "etaLow": "-0.9",
         "etaUp": "0.9",
-        "evSel": "true",
+        "evSel": "false",
```

Comment:
Revert. Event selection should be on by default.

Reply:
But then we have the clear differences in the number of jets (?).